lib/travis/api/v3/billing.rb

```diff
@@ -0,0 +1,56 @@
+module Travis::API::V3
+  class Billing
+    attr_reader :user_id, :subscription_id
+
+    def initialize(user_id = nil, subscription_id = nil)
+      @user_id = user_id
+      @subscription_id   = subscription_id
+    end
+
+    def create_subscription(subscription_params)
+      post('/subscriptions', {user_id: user_id, subscription: subscription_params})
+      #what will Billing return? -> subscription id?
+    end
+
+    def cancel_subscription
+      post("/subscriptions/#{subscription_id}/cancel", {user_id: user_id})
+    end
+
+    def edit_address(address)
+      patch("/subscriptions/#{subscription_id}/address", {user_id: user_id, address: address})
+      #what will Billing return? -> subscription id?
```

Comment:
In all these methods it currently returns an empty body when successful (with `201` status). Would something else be useful? (I guess the id not particularly, you have to have to make the request)